app.js

```diff
@@ -12,7 +13,7 @@ async function bootstrap () {
   let config = { models, stellar }
 
   const adapters = [
-    new Reddit(config)
+    new Slack(config)
```

Comment:
We shouldn't get rid of Reddit :-)

I think it would be nice if we'd have a way to configure which adapters one wants to use via .env files, e.g. `ACTIVATE_REDDIT_ADAPTER=1` and `ACTIVATE_SLACK_ADAPTER=1` and then check for it.

Reply:
I was thinking that perhaps the way you configure which adapters you want is simply by adding them into the adapters array. What do you think of that?